src/test/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/integration/GoalControllerTest.kt

```diff
@@ -642,7 +649,7 @@ class GoalControllerTest : IntegrationTestBase() {
     @Test
     open fun `re-add goal with a note and no target date`() {
       val goalUuid = "31d7e986-4078-4f5c-af1d-115f9ba3722d"
-      val note = Goal(note = "Re-adding note")
+      val note = Goal(note = "Re-adding note", reminderDate = LocalDate.now().plusWeeks(20).toString())
```

Comment:
This reminderDate value is never asserted. Should it be?

Reply:
Yes, added [ed561c7](https://github.com/ministryofjustice/hmpps-sentence-plan/pull/201/commits/ed561c788900d67665d85c4514de31fce8d64b6a)